<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="15" passed="12" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-22T10:25:01 IST" name="Suite" finished-at="2022-08-22T10:25:23 IST" duration-ms="21943">
    <groups>
    </groups>
    <test started-at="2022-08-22T10:25:01 IST" name="RegisterTest" finished-at="2022-08-22T10:25:05 IST" duration-ms="3836">
      <class name="com.automation.Test.RegisterTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.automation.Test.RegisterTest@16a0ee18]" started-at="2022-08-22T10:25:01 IST" name="beforeTest" finished-at="2022-08-22T10:25:02 IST" duration-ms="1360" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:com.automation.Test.RegisterTest@16a0ee18]" started-at="2022-08-22T10:25:02 IST" name="beforeClass" finished-at="2022-08-22T10:25:03 IST" duration-ms="509" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="registerWithValidData()[pri:1, instance:com.automation.Test.RegisterTest@16a0ee18]" started-at="2022-08-22T10:25:03 IST" name="registerWithValidData" finished-at="2022-08-22T10:25:04 IST" duration-ms="1660" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerWithValidData -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.automation.Test.RegisterTest@16a0ee18]" started-at="2022-08-22T10:25:04 IST" name="tearDown" finished-at="2022-08-22T10:25:05 IST" duration-ms="153" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automation.Test.RegisterTest -->
    </test> <!-- RegisterTest -->
    <test started-at="2022-08-22T10:25:05 IST" name="LoginTest" finished-at="2022-08-22T10:25:09 IST" duration-ms="4817">
      <class name="com.automation.Test.LoginTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:05 IST" name="beforeTest" finished-at="2022-08-22T10:25:05 IST" duration-ms="732" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:05 IST" name="beforeClass" finished-at="2022-08-22T10:25:06 IST" duration-ms="346" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="verifyLoginWithInValidData(java.lang.String,java.lang.String)[pri:1, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:06 IST" name="verifyLoginWithInValidData" data-provider="loginProvider" finished-at="2022-08-22T10:25:07 IST" duration-ms="840" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[user1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login Unsuccesful expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login Unsuccesful expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at com.automation.Test.LoginTest.verifyLoginWithInValidData(LoginTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInValidData -->
        <test-method signature="verifyLoginWithInValidData(java.lang.String,java.lang.String)[pri:1, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:07 IST" name="verifyLoginWithInValidData" data-provider="loginProvider" finished-at="2022-08-22T10:25:07 IST" duration-ms="352" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[user2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login Unsuccesful expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login Unsuccesful expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at com.automation.Test.LoginTest.verifyLoginWithInValidData(LoginTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInValidData -->
        <test-method signature="verifyLoginWithInValidData(java.lang.String,java.lang.String)[pri:1, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:07 IST" name="verifyLoginWithInValidData" data-provider="loginProvider" finished-at="2022-08-22T10:25:08 IST" duration-ms="354" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[user3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login Unsuccesful expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login Unsuccesful expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at com.automation.Test.LoginTest.verifyLoginWithInValidData(LoginTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInValidData -->
        <test-method signature="verifyLoginWithValidData()[pri:2, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:08 IST" name="verifyLoginWithValidData" finished-at="2022-08-22T10:25:09 IST" duration-ms="1103" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithValidData -->
        <test-method signature="verifyLogout()[pri:0, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:09 IST" name="verifyLogout" description="verify logout" finished-at="2022-08-22T10:25:09 IST" depends-on-methods="com.automation.Test.LoginTest.verifyLoginWithValidData" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogout -->
        <test-method signature="verifyResetPassword()[pri:0, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:09 IST" name="verifyResetPassword" description="verify password" finished-at="2022-08-22T10:25:09 IST" depends-on-methods="com.automation.Test.LoginTest.verifyLogout" duration-ms="219" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyResetPassword -->
        <test-method signature="verifySignUpLink()[pri:0, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:09 IST" name="verifySignUpLink" description="verify signup" finished-at="2022-08-22T10:25:09 IST" depends-on-methods="com.automation.Test.LoginTest.verifyLogout" duration-ms="162" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySignUpLink -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.automation.Test.LoginTest@31fa1761]" started-at="2022-08-22T10:25:09 IST" name="tearDown" finished-at="2022-08-22T10:25:09 IST" duration-ms="169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automation.Test.LoginTest -->
    </test> <!-- LoginTest -->
    <test started-at="2022-08-22T10:25:10 IST" name="SearchFlightTest" finished-at="2022-08-22T10:25:16 IST" duration-ms="6719">
      <class name="com.automation.Test.SearchFlightTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:10 IST" name="beforeTest" finished-at="2022-08-22T10:25:10 IST" duration-ms="729" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:10 IST" name="beforeClass" finished-at="2022-08-22T10:25:11 IST" duration-ms="343" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="BasicSearch()[pri:1, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:11 IST" name="BasicSearch" finished-at="2022-08-22T10:25:12 IST" duration-ms="1554" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BasicSearch -->
        <test-method signature="ParameterizedSearch()[pri:2, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:12 IST" name="ParameterizedSearch" finished-at="2022-08-22T10:25:13 IST" duration-ms="997" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ParameterizedSearch -->
        <test-method signature="RoundTripSearch()[pri:3, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:13 IST" name="RoundTripSearch" finished-at="2022-08-22T10:25:15 IST" duration-ms="1570" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RoundTripSearch -->
        <test-method signature="MultiCitySearch()[pri:4, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:15 IST" name="MultiCitySearch" finished-at="2022-08-22T10:25:16 IST" duration-ms="1350" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MultiCitySearch -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.automation.Test.SearchFlightTest@27f981c6]" started-at="2022-08-22T10:25:16 IST" name="tearDown" finished-at="2022-08-22T10:25:16 IST" duration-ms="163" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automation.Test.SearchFlightTest -->
    </test> <!-- SearchFlightTest -->
    <test started-at="2022-08-22T10:25:16 IST" name="BookTest" finished-at="2022-08-22T10:25:23 IST" duration-ms="6281">
      <class name="com.automation.Test.BookingTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.automation.Test.BookingTest@2d710f1a]" started-at="2022-08-22T10:25:16 IST" name="beforeTest" finished-at="2022-08-22T10:25:17 IST" duration-ms="740" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:com.automation.Test.BookingTest@2d710f1a]" started-at="2022-08-22T10:25:17 IST" name="beforeClass" finished-at="2022-08-22T10:25:17 IST" duration-ms="329" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="turnOffWifi()[pri:1, instance:com.automation.Test.BookingTest@2d710f1a]" started-at="2022-08-22T10:25:17 IST" name="turnOffWifi" finished-at="2022-08-22T10:25:20 IST" duration-ms="2568" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- turnOffWifi -->
        <test-method signature="loadSearchPage()[pri:0, instance:com.automation.Test.BookingTest@2d710f1a]" started-at="2022-08-22T10:25:20 IST" name="loadSearchPage" finished-at="2022-08-22T10:25:21 IST" depends-on-methods="com.automation.Test.BookingTest.turnOffWifi" duration-ms="1236" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadSearchPage -->
        <test-method signature="verifyBooking()[pri:0, instance:com.automation.Test.BookingTest@2d710f1a]" started-at="2022-08-22T10:25:21 IST" name="verifyBooking" finished-at="2022-08-22T10:25:22 IST" depends-on-methods="com.automation.Test.BookingTest.loadSearchPage" duration-ms="1250" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBooking -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.automation.Test.BookingTest@2d710f1a]" started-at="2022-08-22T10:25:22 IST" name="tearDown" finished-at="2022-08-22T10:25:23 IST" duration-ms="146" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automation.Test.BookingTest -->
    </test> <!-- BookTest -->
  </suite> <!-- Suite -->
</testng-results>
